google/cloud/bigquery/v2/minimal/internal/job_request_test.cc

```diff
@@ -177,6 +177,32 @@ TEST(ListJobsRequestTest, OutputStream) {
   EXPECT_EQ(expected, os.str());
 }
 
+TEST(GetJobRequest, DebugString) {
+  GetJobRequest request("test-project-id", "test-job-id");
+  request.set_location("test-location");
+  std::string expected = absl::StrCat(
+      "GetJobRequest{project_id=", request.project_id(),
+      ", job_id=", request.job_id(), ", location=", request.location(), "}");
```

Comment:
optional: there is something strange about writing the same code again in a test.

Consider writing the expected string in full:

```cc
std::string expected =
     "GetJobRequest{project_id=test-project-id, job_id=test-job-id, "
     "location=test-location}";
```